cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -499,7 +512,7 @@ describe('Script Builder', () => {
           cy.getByTestID('flux-sync--toggle').should('have.class', 'disabled')
         })
 
-        describe('using hotkeys:', () => {
+        it('using hotkeys:', () => {
```

Comment:
Specifically, the biggest gain was in the lack of user provisioning. I got it down to 5 minutes without combining the hotkeys test. The added time saving from the hotkeys test combination, was ~ 30 seconds.

I would rather not combine more tests together, unless we really need to shave off more time.

Because there is a tradeoff on combining too many tests into a single block. When we combine, then we loose the ability to see exactly which test (which `it()` block) fails in circleci. Such as with here:
<img width="282" alt="Screen Shot 2022-10-19 at 1 30 46 PM" src="https://user-images.githubusercontent.com/10232835/196797745-f1086a8c-4a8d-42e3-b432-53d252cd0c7b.png">

That ^^ message has scoped the failing to exactly which it test is flaky. Otherwise we have to look at the video artifacts (not available in firefox) or the error messages itself -- which may (or may not) helps us pinpoint the exact issue.

Reply:
Got it! Thanks for the explanation!